packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -600,32 +599,34 @@ export const GiantTrackTile = ({
               {renderStatsRow()}
             </div>
 
-            <div
-              className={cn(styles.actionButtons, fadeIn)}
-              role='group'
-              aria-label={messages.actionGroupLabel}
-            >
-              {renderShareButton()}
-              {renderMakePublicButton()}
-              {hasStreamAccess && renderRepostButton()}
-              {hasStreamAccess && renderFavoriteButton()}
-              <span>
-                {/* prop types for overflow menu don't work correctly
+            {isUnlisted && !isOwner ? null : (
```

Comment:
i could put this logic inside the `render` functions but we also want to hide the overflow menu. open to ideas.

Reply:
I think this is fine

nit: we generally do the null at the end like `condition ? <Content /> : null`
I see both in the codebase, but this one is about 6 times as frequent 🤷